apps/userdashboard/templates/a4_candy_userdashboard/base_userdashboard.html

```diff
@@ -135,11 +135,11 @@ <h4>{% trans 'Organisations' %}</h4>
                         {% for organisation in view.organisations %}
                         <a class="btn btn--link text-decoration-none text-muted d-flex align-items-center" href="{% url 'organisation' organisation_slug=organisation.slug %}">
                           {% if organisation.logo %}
-                            <img src="{% thumbnail organisation.logo '0x40' %}" alt="{{ organisation.name }}" />
+                            <img src="{% thumbnail organisation.logo '0x40' %}" alt="{{ organisation.title }}" />
                           {% else %}
-                            <i class="fas fa-home font--lg pt-1" aria-label="{{ organisation.name }} {% trans 'Home' %}"></i>
+                            <i class="fas fa-home font--lg pt-1" aria-label="{{ organisation.title }} {% trans 'Home' %}"></i>
                           {% endif %}
-                            &nbsp {{ organisation.name }}
+                            &nbsp {{ organisation.title }}
```

Comment:
`organisation.title` is empty, `organisation.name` shows the org name

Reply:
have you given your oganisation a title? the name comes from the django admin and is normally not as human readable, the title comes from the organisation settings in the dashboard

Reply:
i see, my bad